ui/blocks/BlocksTable.tsx

```diff
@@ -46,9 +46,10 @@ const BlocksTable = ({ data, isLoading, top, page, showSocketInfo, socketInfoNum
           <Th width="64px" isNumeric>Txn</Th>
           <Th width={ `${ GAS_COL_WEIGHT / widthBase * 100 }%` }>Gas used</Th>
           { !isRollup && !config.UI.views.block.hiddenFields?.total_reward &&
+              // DOUBLE CHECK: we have this other Reward column, is it ok?
               <Th width={ `${ REWARD_COL_WEIGHT / widthBase * 100 }%` }>Reward { config.chain.currency.symbol }</Th> }
           { !isRollup && !config.UI.views.block.hiddenFields?.burnt_fees &&
-              <Th width={ `${ FEES_COL_WEIGHT / widthBase * 100 }%` }>Burnt fees { config.chain.currency.symbol }</Th> }
+              <Th width={ `${ FEES_COL_WEIGHT / widthBase * 100 }%` }>Reward pool fees { config.chain.currency.symbol }</Th> }
```

Comment:
What is the difference, i.e. is the Reward column empty?

I'd like to compare the data displayed in the redesign w the current explorer and make sure they are the same for the Reward pool fees colum.

Reply:
I haven't seen any example of this, but after doing some reading, my understanding is that in some blockchain systems, fees are typically divided into two parts: rewards (portion of the fees allocated as rewards to validators/miners) and burnt fees (portion of the fees that are removed from circulation), and those are the parts here, that tipically are different but in our case we use the burnt fees as rewards.

I'm not sure why we use burnt fees as rewards, if the system has a dedicated field for them, but as we're considering burnt fees as rewards, I think we should be good to remove the reward column.

What do you think @yurikoinaba ?